code/modules/mob/living/carbon/life.dm

```diff
@@ -41,6 +41,8 @@
 		return
 	if(NO_BREATH in mutations)
 		return // No breath mutation means no breathing.
+	if(istype(src, /mob/living/carbon/brain))
```

Comment:
This is bad, you don't put istype checks for a subtype inside the proc of a parent type. Just have `/mob/living/carbon/brain/breathe()` return instead of doing this.